src/Microsoft.Performance.SDK.Runtime/Extensibility/DataExtensions/DataCookers/CompositeDataCookerReference.cs

```diff
@@ -92,7 +91,7 @@ private CompositeDataCookerReference(Type type)
 
             reference = null;
 
-            if (candidateType.IsPublicAndInstantiatableOfType(typeof(ICompositeDataCookerDescriptor)))
+            if (candidateType.IsInstantiatableOfType(typeof(ICompositeDataCookerDescriptor)))
```

Comment:
what's the reason for this change?

Reply:
It's required to support non-public custom data source, which is something we want specifically for DataLayer, but would also be good to have in general.